app/models/spree/volume_price.rb

```diff
@@ -14,6 +14,16 @@ class Spree::VolumePrice < ActiveRecord::Base
 
   validate :range_format
 
+  def self.for_variant(variant, user: nil)
```

Comment:
It's very well covered by the pricer spec and I already have 100% coverage of the `volume_price` class. I will add one if you insist, though.